DXMainClient/DXGUI/Generic/CampaignSelector.cs

```diff
@@ -28,18 +32,23 @@ public class CampaignSelector : XNAWindow
             "INI/Map Code/Difficulty Hard.ini"
         };
 
-        public CampaignSelector(WindowManager windowManager, DiscordHandler discordHandler) : base(windowManager)
+        public CampaignSelector(WindowManager windowManager, DiscordHandler discordHandler, CampaignTagSelector campaignTagSelector) : base(windowManager)
         {
             this.discordHandler = discordHandler;
+            this.campaignTagSelector = campaignTagSelector;
         }
 
         private DiscordHandler discordHandler;
+        private CampaignTagSelector campaignTagSelector;
 
-        private List<Mission> Missions = new List<Mission>();
+        private List<Mission> lbCampaignListMissions = new List<Mission>();
```

Comment:
this is not listbox, hence the `lb` prefix is incorrect

Reply:
82ac5130 Rename lbCampaignListMissions to selectedMissions